pyshelf/bulk_update/utils.py

```diff
@@ -23,24 +58,41 @@ def run(args):
     # a no-op and its easiest to use it in the subprocesses
     # because then it also automatically configures child
     # loggers such as boto and elasticsearch
-    handler = logging.StreamHandler(sys.stdout)
-    logger = logging.getLogger("update-search-index")
+    handler = _get_logging_handler(logger_name, args.get("--log-directory"))
+    logger = logging.getLogger(logger_name)
     logger.addHandler(handler)
     logger.setLevel(log_level)
 
     config = {
-        "logLevel": log_level,
-        "chunkSize": int(args["--chunk-size"])
+        "logLevel": log_level
     }
 
+    if args.get("--chunk-size"):
```

Comment:
This I also would be prefer not to have to do.  We should split out this functionality more thoroughly if we want to use this.